### ISSUE TITLE
Parameter namespace is being ignored in Shell

### ISSUE THREAD

**Issue by viniciusartur** [AvitalTamir/cyphernetes#159]
Given I changed the namespace of my kubectl context:
```
❯ kubectl config set-context --current --namespace=test                      ─╯
Context "default" modified.

❯ kubectl config get-contexts                                                ─╯
CURRENT   NAME      CLUSTER   AUTHINFO   NAMESPACE
*         default   default   default    test
```

When I explicitly provide the argument `--namespace default` to the shell interface as in:
```
./cyphernetes shell --namespace default
```
The argument is being ignored and the one from the kubectl context is being used instead:
```
(default) test »
```

The namespace from the kubectl context is always taking precedence instead of the one explicitly provided by the user.

I think the namespace from the kubectl context should take precedence unless the user explicitly provides a namespace via the parameter `--namespace`.

In other words:
 - if the kubectl context has namespace `test` and no `--namespace <NAMESPACE>` is provided, then use the namespace `test`
 - if the kubectl context has namespace `test`, but user provided `--namespace someother`, then user the namespace `someother`

**Comment by viniciusartur** [comment]
I would like to work on this issue. If it's possible, please assign it to me. Thanks!

**Comment by AvitalTamir** [comment]
@viniciusartur good catch! Thanks for opening the issue and willing to do this, assigned to you :)

**Comment by viniciusartur** [comment]
I submitted the PR https://github.com/AvitalTamir/cyphernetes/pull/161 to fix this issue.
I also checked that this issue is not happening for `query` and `web` commands.

**Comment by AvitalTamir** [comment]
Thanks for the contribution @viniciusartur this is great 👑 
Merged!
